src/main/scheduler/scheduler.c

```diff
@@ -442,10 +442,23 @@ FAST_CODE void scheduler(void)
     uint32_t nextTargetCycles = 0;
     int32_t schedLoopRemainingCycles;
 
+    nowCycles = getCycleCounter();
+    currentTimeUs = clockCyclesToMicros(nowCycles);
+
+#ifdef USE_RX_EXPRESSLRS
```

Comment:
Having this ahead of the gyro task introduces jitter in the reading of the gyro and application of the filters. Putting this between the filter task and the PID task may be more acceptable from a timing perspective. I don’t have the hardware to test such a change so please try that and report back. Thanks.

Reply:
You, noticed that, I will look into it.

Reply:
Actually, even that is not really be happy with. Do you know what bit of code isn’t being called frequently enough? What would be acceptable is to add a small amount of processing to the rx check routine, or even create another task to handle this, but please don’t mess with the scheduler itself.

Reply:
@SteveCEvans below code block is sensitive to timing. `fhssRequired` flag is set in the hardware timer IRQ and it needs to be executed right after. It's responsible for hopping to the next channel or sending the telemetry packet.
https://github.com/betaflight/betaflight/blob/master/src/main/rx/expresslrs.c#L1066-L1074
I'll experiment with creating a small task that can run this code more often.